SRC/classes/CountryInformation.cls

```diff
@@ -0,0 +1,117 @@
+        mapOfCountry.put('+1',americaMap);
+        
+        // Find The Combination Of Country State and District
+        for(String countryCode : mapOfCountry.keySet()){
+        //Get Country Name   
+            Map<String,Map<String,List<String>>> countriesName = mapOfCountry.get(countryCode); 
+
+            for(String country : countriesName.keySet()){
+            
+                if(country.startsWith('S')){
+                //Get State Name   
+                   Map<String,List<String>> stateNames = countriesName.get(country);
+                    
+                    for(String state : stateNames.keySet()){
+                                                 
+                        if(state.startsWith('A')){
+                        //Get District Name    
+                            List<String> districtList = stateNames.get(state);
+                             
+                            for(String district : districtList){
+                                 
+                                if(district.startsWith('S')&&district.length()>=5){
+                                    System.debug('Country Name  : ' + country);
+                                    System.debug('State Name : ' + state);
+                                    System.debug('District Name  : ' + district);
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+       
+        // Finding  Districts which length is more or equal to some number
+        List<String> allDistricts = new List<String>();
+        for(String countryCode : mapOfCountry.keySet()){
+    
+            Map<String,Map<String,List<String>>> countriesName = mapOfCountry.get(countryCode);
+            for(String country : countriesName.keySet()){
+                // get state map
+                Map<String,List<String>> stateNames = countriesName.get(country); 
+                for(String state : stateNames.keySet()){
+                    // get district name list 
+                    List<String> districtList = stateNames.get(state);
+                        // add all district in districtList
+                        allDistricts.addAll(districtList);
+                }
+            }
+        }
+
+        // get districts from all district list
+        for(String districtLength : allDistricts){
+            // cheacking district length 
+            if(districtLength.length()>=1 && districtLength.length()>=5){
+                // print the district  given length
+                System.debug('District of Given Length : ' +districtLength);
+            }
+        }
+    }
+}
+
+/*
+:::: --- >>> How to run the code in developer console in org
+1. Firstly we need to write our class Name in anonyomous window.
+2. Then simply use the (.) dot notation to access the class method.
+
+Running command for current program - CountryInformation.insertDataValues();
```

Comment:
Hi can you also add what code you added in anonymous window here, so I can understand what is the output and compare is it correct or not.